doc/languages-frameworks/javascript.section.md

```diff
@@ -258,34 +258,79 @@ It returns a derivation with all `package-lock.json` dependencies downloaded int
 
 #### importNpmLock {#javascript-buildNpmPackage-importNpmLock}
 
-`importNpmLock` is a Nix function that requires the following optional arguments:
+This function replaces the npm dependency references in `package.json` and `package-lock.json` with paths to the Nix store.
+How each dependency is fetched can be customized with the `fetcherOpts` argument.
 
-- `npmRoot`: Path to package directory containing the source tree
+This is simpler and more convenient alternative to [`fetchNpmDeps`](#javascript-buildNpmPackage-fetchNpmDeps) for managing npm dependencies in Nix.
+There is no need to specify a `hash`; since it relies entirely on the integrity hashes already present in the `package-lock.json` file.
```

Comment:
```suggestion
There is no need to specify a `hash`, since it relies entirely on the integrity hashes already present in the `package-lock.json` file.
```